components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.experiments.debug
+
+import android.app.Activity
+import android.os.Bundle
+import android.support.annotation.VisibleForTesting
+import mozilla.components.service.experiments.Experiments
+import mozilla.components.support.base.log.logger.Logger
+
+class ExperimentsDebugActivity : Activity() {
+    private val logger = Logger(this.javaClass.canonicalName)
+
+    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
+    internal var testExperimentsUpdated = false
```

Comment:
nit: I don't personally like introducing these kind of variables to make the test work, but I have no concrete suggestion on how to make this better, so ... yeah :D

Reply:
I didn't like it either, but I couldn't come up with a better way at the time either.

Reply:
I managed to come up with a better way to resolve this since I ran into a problem with the debug activity attempting to do networking on the main thread.